<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaniKaniWidget</class>
 <widget class="QWidget" name="WaniKaniWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaniKani</string>
  </property>
  <layout class="QVBoxLayout" name="layout" stretch="1,0,0">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <item>
        <layout class="QVBoxLayout" name="settingsVerticalLayout">
         <item>
          <layout class="QHBoxLayout" name="apiKeyLayout">
           <item>
            <widget class="QLabel" name="apiKeyLabel">
             <property name="text">
              <string>API key:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="apiKeyValue"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="userInformationGroupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="userInformationLayout">
            <item>
             <widget class="QLabel" name="gravatarValue">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="userInformationValue">
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="wallpaperGroupBox">
           <property name="title">
            <string>Wallpaper</string>
           </property>
           <layout class="QVBoxLayout" name="wallpaperLayout">
            <item>
             <layout class="QHBoxLayout" name="refreshLayout">
              <item>
               <widget class="QSpinBox" name="intervalSpinBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="suffix">
                 <string> min</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>60</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="forceUpdateButton">
                <property name="text">
                 <string>Force Update</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="kanjisLayout">
              <item>
               <widget class="QRadioButton" name="currentKanjisRadioButton">
                <property name="text">
                 <string>Current Kanjis</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="allKanjisRadioButton">
                <property name="text">
                 <string>All Kanjis</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontComboBox">
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="writingSystem">
               <enum>QFontDatabase::Japanese</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="fontStylesLayout">
              <item>
               <widget class="QCheckBox" name="boldFontCheckBox">
                <property name="text">
                 <string>Bold</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="italicsFontCheckBox">
                <property name="text">
                 <string>Italics</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="colorsLayout">
              <item row="7" column="1" colspan="2">
               <widget class="QPushButton" name="swapPushButton">
                <property name="text">
                 <string>Swap</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="pendingLabel">
                <property name="text">
                 <string>Pending</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pendingForegroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="pendingBackgroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="backgroundLabel">
                <property name="text">
                 <string>Background</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="apprenticeLabel">
                <property name="text">
                 <string>Apprentice</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="apprenticeBackgroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QPushButton" name="masterBackgroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="guruForegroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="apprenticeForegroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="masterLabel">
                <property name="text">
                 <string>Master</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QPushButton" name="masterForegroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="enlightenedLabel">
                <property name="text">
                 <string>Enlightened</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QPushButton" name="burnedForegroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QPushButton" name="guruBackgroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QPushButton" name="enlightenedBackgroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="enlightenedForegroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="guruLabel">
                <property name="text">
                 <string>Guru</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QPushButton" name="burnedBackgroundPushButton">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="burnedLabel">
                <property name="text">
                 <string>Burned</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="foregroundLabel">
                <property name="text">
                 <string>Foreground</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="settingsHorizontalLayout">
         <item>
          <widget class="QPushButton" name="resetAllPushButton">
           <property name="text">
            <string>Reset All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout" stretch="1,0">
     <item>
      <widget class="QLabel" name="aboutValue">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="closeLayout">
       <item>
        <spacer name="closeVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="closeToolButton">
         <property name="toolTip">
          <string>Quit</string>
         </property>
         <property name="icon">
          <iconset resource="../res/resources.qrc">
           <normaloff>:/close</normaloff>:/close</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
